packages/api/src/utils/client/fetch.ts

```diff
@@ -5,6 +5,7 @@
  */
 async function wrappedFetch(url: string | URL, init?: RequestInit): Promise<Response> {
   try {
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
Please add a comment here which explains why disabling this rule is important here. I believe that's the only place we need to use native fetch.

Reply:
I added a comment but obviously the function can't call itself, so I thought it is kinda self explanatory here